include/tvm/relay/expr_functor.h

```diff
@@ -197,7 +198,7 @@ class ExprMutator
    * \brief Mutate is alias for VisitExpr
    * \return expr.
    */
-  Expr Mutate(const Expr& expr) {
+  virtual Expr Mutate(const Expr& expr) {
```

Comment:
Can directly overload VisitExpr, no need to overload mutate

Reply:
:+1: